### PR TITLE
Enable release nugets in microupdate-3.2 branch

### PR DESCRIPTION
**Pull request by shyamnamboodiripad**
Port #12019 from master to microupdate-3.2 branch so that we can produce release nugets as part of build in the microupdate-3.2 branch

### CODE REVIEW

**Comment by shyamnamboodiripad** [comment]
@cston @jasonmalinowski @tmat @ManishJayaswal 

@MattGertz for approval

FYI @dotnet/roslyn-infrastructure

**Comment by shyamnamboodiripad** [comment]
Ping @jasonmalinowski @tmat @cston 

(This just makes it so that we can generate release nugets in the MU3.2 branch as we do in master currently - this is a simple port of the changes already in master so there shouldn't be anything new to review here :))

File: build/BuildNuGets.csx [hunk 1]
```diff
@@ -186,44 +214,50 @@ void GeneratePublishingConfig(string fileName, IEnumerable<XElement> packages)
...
-
 Directory.CreateDirectory(OutDir);
+var ErrorLogFile = Path.Combine(OutDir, "skipped_packages.txt");
+try
+{
+    if (File.Exists(ErrorLogFile)) File.Delete(ErrorLogFile);
+}
+catch
+{
+    // Ignore errors
```
Comment (jasonmalinowski) [inline build/BuildNuGets.csx]: Why? This seems bad. :-)

**Comment by jasonmalinowski** [comment]
:+1:

File: build/BuildNuGets.csx (hunk 1 above)
Comment (shyamnamboodiripad) [inline build/BuildNuGets.csx]: Agreed - will remove later when I update the script in master.

**Pull request merged**